Assets/MixedRealityToolkit/Inspectors/MixedRealityStandardShaderGUI.cs

```diff
@@ -1018,5 +1063,49 @@ protected static void SetColorProperty(Material material, string propertyName, C
                 material.SetColor(propertyName, propertyValue.Value);
             }
         }
+
+        [MenuItem("Mixed Reality Toolkit/Utilities/Upgrade MRTK Standard Shader for Lightweight Render Pipeline")]
+        protected static void UpgradeShaderForLightweightRenderPipeline()
```

Comment:
I'd like to make sure we have a plan to communicate this out to our devs that they need to do this. Cameron, since you made the PR, could you please be on point to communicate out that people may need to click this upgrade button? You could work with @wiwei  since he is managing the release

Reply:
Of course! I agree we should get this in the release notes and maybe bubble it up higher in the docs. Unity has a similar [obfuscated menu item ](https://docs.unity3d.com/Packages/com.unity.render-pipelines.lightweight@4.0/manual/images/MaterialUpgraders.png) for upgrading a project. I thought about adding the MRTK menu item there too, but Unity keeps changing the location with each release.

Reply:
Maybe once we make the Lightweight Scriptable Render Pipeline is more standard (when we upgrade to Unity 2019+) we can build some tooling that auto detects shaders need to be upgraded? But, Unity doesn't even do this. You just end up with a bunch of magenta materials.